ddtrace/contrib/asgi/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+The asgi__ middleware for tracing all requests to an ASGI-compliant application.
+
+To configure tracing manually::
```

Comment:
I think we can be straight to the point here.

```rst
To configure tracing manually::

    from ddtrace.contrib.asgi import TraceMiddleware

    # app = <your asgi app>
    app = TraceMiddleware(app)
```

Reply:
Added on [address pr comments](https://github.com/DataDog/dd-trace-py/pull/1567/commits/07e56e8d7b88158c13f370163400d89fdcc28a8b).